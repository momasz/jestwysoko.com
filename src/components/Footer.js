import React from "react";

const Footer = () => (
  <div className={"footer"}>
    <div className={"container"}>
      <p>jestwysoko.com ðŸ¤˜</p>
    </div>
  </div>
);

export default Footer;
