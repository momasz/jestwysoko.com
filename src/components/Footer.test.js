import React from 'react';
import { mount } from 'enzyme';
import Footer from './Footer';

describe('Footer component', () => {
  it('renders as expected', () => {
    const footer = mount(
      <Footer/>
    );
    expect(footer).toMatchSnapshot();
  });
});
