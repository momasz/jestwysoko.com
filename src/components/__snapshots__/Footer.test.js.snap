// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders as expected 1`] = `
<Footer>
  <p>
    jestwysoko.com 
    <strong>
      Â©
    </strong>
     2020
  </p>
</Footer>
`;
